logic/src/commonMain/kotlin/com/wire/kalium/logic/data/call/CallRepository.kt

```diff
@@ -356,7 +363,7 @@ internal class CallDataSource(
         }
     }
 
-    override fun updateCallParticipants(conversationId: String, participants: List<Participant>) {
+    override fun updateCallParticipants(conversationId: String, participants: List<Participant>, scope: CoroutineScope) {
```

Comment:
Ugly having to pass the calling scope here. Any reason why we can't have our own calling scope inside the `CallRepostitory`?

Or maybe we inject the same scope into CallMananger and CallRepository?

cc @ohassine @alexandreferris